### ISSUE TITLE
Rake task(s) for regenerating example reports.

### ISSUE THREAD

**Issue by duckinator** [how-is/how_is#84]
Create Rake task(s) for regenerating example reports.

```
$ rake generate_example_report:html
# Re-generate example HTML report.
$ rake generate_example_report:json
# Re-generate example JSON report.
$ rake generate_example_report:all
# Re-generate all reports.
$
```

**Comment by duckinator** [comment]
https://edelpero.svbtle.com/everything-you-always-wanted-to-know-about-writing-good-rake-tasks-but-were-afraid-to-ask possibly useful

**Comment by duckinator** [comment]
Closed by #86.
